### ISSUE TITLE
Considering dropping Opera from Firefox Platform Status

### ISSUE THREAD

**Issue by takenspc** [mozilla/platform-status#605]
While I respect great standardization and implementation works from Opera, it is hard to track Opera's platform status:

- there is no official platform status (while Edge continues to maintain its platform status)
- though Firefox Platform Status fills in data from Can I use..., not every feature has Can I use... entry 

As a result, Opera's position at Firefox Platform Status looks very inconsistent. At Firefox Platform Status, Opera looks like not to support Asynchronous Clipboard API, Web Animations API and many other features.

In general speaking, Opera's platform status is considered same as Chrome's one. There are rarely occasions when it is necessary to make a distinction between them.

Thus, I propose that dropping Opera from Firefox Platform Status until Opera starts to maintain its platform status.

**Comment by digitarald** [comment]
Agreed, they don't seem to actively invest in the platform as much anymore and/or doing outreach to devs.

**Comment by digitarald** [comment]
@chrisdavidmills what's the compat data view on opera?

**Comment by chrisdavidmills** [comment]
@digitarald we have compat data for Opera and Opera mobile on our compat data repo, but generally it is just worked out from the chrome version number/status. I don't think we'd miss a lot from removing it; we could perhaps add something to indicate that Chrome support also implies support for the other chromium-based browsers, although this is fairly obvious to any web dev.

**Comment by digitarald** [comment]
> we could perhaps add something to indicate that Chrome support also implies support for the other chromium-based browsers, although this is fairly obvious to any web dev.

Right, same for Samsung, Edge Chromium, Brace, etc. Keeping track of them all or even picking one isn't great. Maybe changing the icon to a more general Chromium icon can also clarify that.
